sdk/python/pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[tool.poetry]
+name = 'symbol-sdk-python'
```

Comment:
dropped core, but problematic in that we need obsoletes somehow 🤔

Reply:
as I've mentioned, I think obsoletes is only needed _once_ when releasing initial package that replaces different package (and there is NAME change involved), besides that obsoletes should not be needed)